report_thesis/src/sections/experiment_design/initial_experiment.tex

```diff
@@ -0,0 +1,18 @@
+\subsection{Design for Initial Experiment}\label{sec:initial-experiment}
+The goal of the initial experiment is to identify the most promising models for further evaluation.
```

Comment:
Past tense.

```suggestion
The goal of the initial experiment was to identify the most promising models for further evaluation.
```

Reply:
Should addressed now that the intro was changed completely.